config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -664,6 +664,40 @@ postsubmits:
       - name: creds
         secret:
           secretName: deployer-service-account
+  - name: post-test-infra-upload-testgrid-config
+    cluster: test-infra-trusted
+    branches:
+    - master
+    labels:
+      preset-bazel-scratch-dir: "true"
+    env:
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: /etc/service-account/service-account.json
```

Comment:
I don't think the env is required since you are using a flag now.

Reply:
/hold
I'm pretty sure you're right, but I want to make sure

Reply:
Writing to TestGrid canary was successful using `pj-on-kind.sh`

```
INFO: Running command line: bazel-bin/testgrid/cmd/configurator/linux_amd64_pure_stripped/configurator '--gcp-service-account=/etc/service-account/service-account.json' '--yaml=/home/prow/go/src/github.com/chases2/test-infra/config/testgrids' '--default=/home/prow/go/src/github.com/chases2/test-infra/config/testgrids/default.yaml' '--prow-config=/home/prow/go/src/github.com/chases2/test-infra/prow/config.yaml' '--prow-job-config=/home/prow/go/src/github.com/chases2/test-infra/config/jobs/' '--output=gs://k8s-testgrid-canary/config' --oneshot --world-readable
```

/hold cancel